tox.ini

```diff
@@ -26,6 +27,16 @@ commands =
       --cov-report xml \
       {posargs}
 
+[testenv:cog]
+base_python = python3.13
```

Comment:
We can't use 3.14 yet because one transitive dependency is pydantic, which doesn't yet support 3.14.

They added initial support in https://github.com/pydantic/pydantic/issues/11613 and have released an [alpha](https://github.com/pydantic/pydantic/releases/tag/v2.12.0a1), but we're waiting for a final release of [pydantic 2.12](https://github.com/pydantic/pydantic/milestone/22).